packages/muelu/test/scaling/MatrixLoad.hpp

```diff
@@ -205,6 +207,12 @@ void MatrixLoad(Teuchos::RCP<const Teuchos::Comm<int> >& comm, Xpetra::Underlyin
     comm->barrier();
   }
 
+  if (!blockNumberFile.empty()) {
+    RCP<TimeMonitor> tm = rcp(new TimeMonitor(*TimeMonitor::getNewTimer("Driver: 1f - Read block number")));
+    blocknumber         = Teuchos::rcp_dynamic_cast<LOVector>(Xpetra::IO<SC, LO, GO, Node>::ReadMultiVectorLO(blockNumberFile, map));
```

Comment:
Hm, I'm not sure how I feel about that. I'll let the testing run, but I think we should probably instantiate `Xpetra::IO` for ordinal vectors.